README.md

```diff
@@ -2,322 +2,346 @@
 
 [secret-stack](https://github.com/ssbc/secret-stack) plugin which provides storing of valid secure-scuttlebutt messages in an append-only log.
 
-## What does it do?
+## Table of contents
 
-`ssb-db` provides tools for dealing with unforgeable append-only message 
-feeds. You can create a feed, post messages to that feed, verify a feed created by
-someone else, stream messages to and from feeds, and more (see [API](#api)).
+[What does it do?](#what-does-it-do) | [Example](#example) | [Concepts](#concepts) | [API](#api) | [Stability](#stability)  | [License](#License)
+___
 
-"Unforgeable" means that only the owner of a feed can modify that feed, as
-enforced by digital signing (see [Security properties](#security-properties)).
-This property makes ssb-db useful for peer-to-peer applications. ssb-db also
-makes it easy to encrypt messages.
+# What does it do?
 
-## Example
+`ssb-db` provides tools for dealing with unforgeable append-only message feeds. You can create a feed, post messages to that feed, verify a feed created by someone else, stream messages to and from feeds, and more (see [API](#api)).
 
-In this example, we create a feed, post a signed message to it, then create a stream 
-that reads from the feed.
+*Unforgeable* means that only the owner of a feed can modify that feed, as enforced by digital signing (see [Security properties](#security-properties)).
+
+This property makes `ssb-db` useful for peer-to-peer applications. `ssb-db` also makes it easy to encrypt messages.
+
+# Example
+
+In this example, we create a feed, post a signed message to it, then create a stream that reads from the feed.
 
 ``` js
 /**
  * create an ssb-db instance and add a message to it.
  */
+var pull = require('pull-stream')
 
 //create a secret-stack instance and add ssb-db, for persistence.
 var createApp = require('secret-stack')({})
   .use(require('ssb-db'))
 
-
 // create the db instance.
 // Only one instance may be created at a time due to os locks on port and database files.
 
-var app = createApp(require('ssb-config'))
+var sbot = createApp(require('ssb-config'))
```

Comment:
I kind of thought there was a trend ...away... from using `sbot` as the name for these references. Was a there specific conversation or rationale for replacing it with sbot again here? 

I could be unawares of other perspectives, or the latest chats.

Reply:
Thanks for the incredible feedbak @Connoropolous ! It really is a very long and complex readme. I was sure there must be more than one typo. 
___
### About using sbot or app
I read https://github.com/ssbc/ssb-db/pull/270, I understand it's a discussion that's already come to a conclusion. But the idea was to achieve some consistency between the documentation of the different packages. I remove my commit now and then reopen that discussion.